packages/extension-browser/scripts/import-hints.js

```diff
@@ -0,0 +1,48 @@
+const writeFile = require('fs').writeFile;
+const package = require('../package.json');
+
+const hintModules = Object.keys(package.devDependencies).filter((name) => {
+    return name.startsWith('@hint/hint-');
+});
+
+// Generate hints.import.ts
+
+const hints = `// autogenerated by scripts/import-hints.js
+
+import { IHintConstructor } from 'hint/dist/src/lib/types';
+
+const hints = [
+${
+    hintModules.map((name) => {
+        return `    ...Object.values<IHintConstructor>(require('${name}'))`;
+    }).join(',\n')
+}
+];
+
+export default hints;
+`;
+
+writeFile(`${__dirname}/../src/shared/hints.import.ts`, hints, (err) => {
+    console.error(err);
```

Comment:
I'd a bit more output. Right now the execution of the script outputs which let me 🤔:

```
null
null
```

Maybe do:

```suggestion
    if (err) {
        console.error(err);
    } else {
        console.log(`Created: ${__dirname}/../src/shared/hints.import.ts`);
    }
```
Or just the check for `if (err) { }`.